src/chains/ethereum/ethereum/src/provider-events.ts

```diff
@@ -75,6 +76,11 @@ export type VmAfterTransactionEvent = {
   readonly context: EvmStepContext;
 };
 
+export type VmConsoleLogEvent = {
+  readonly context: EvmStepContext;
+  readonly logs: ConsoleLogs | null;
```

Comment:
Is it not the case that `logs` should be of type `ConsoleLogs`, as we check whether `logs`?
```
if (logs) this.emit("ganache:vm:tx:console.log", { context, logs });
```

https://github.com/trufflesuite/ganache/pull/3327/files#diff-32b463201dcf328f35f745f3b3866381848e899dda327b4d23e4cd1b0ee37db2R456

Reply:
Good catch. Fixed!